scripts/core/device.yml

```diff
@@ -787,14 +787,15 @@ returns:
         - "If the adapter has no underlying equivalent handle."
 --- #--------------------------------------------------------------------------
 type: function
-desc: "Returns synchronized Host and Device global timestamps."
+desc: "Returns a reasonably synchronized Host and Device global timestamps."
```

Comment:
what "reasonably" means here? are we adding that because L0 or other adapter is returning synchronized timestamps with some jitter?

Reply:
This is a small wording change aimed at managing expectations of the accuracy of this entrypoint in UR. I took the wording from the OpenCL spec.  OpenCL is the only adapter which has native support for an equivalent entrypoint. And, even OpenCL does not provide guarantees on the accuracy of the synchronization (hence the "reasonably" wording). I think that the UR spec should be relaxed to at least be similar in scope to OpenCL. I don't think we will ever be able to enforce hard constraints on this entrypoint for CUDA / HIP and probably L0 as well?

This PR is marked as draft because I'm not sure what to do with the existing CTS testing. As it stands, OpenCL is probably the only adapter that can pass this reliably.